python/ray/serve/tests/unit/test_deployment_state.py

```diff
@@ -265,51 +280,6 @@ def check_health(self):
         return self.healthy
 
 
-class MockDeploymentScheduler:
```

Comment:
Killing this to make sure we actually run production one in our tests

Reply:
hallelujah